sdk/lib/rtl/runonce.c

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
may need some "header" telling this comes from wine.

Reply:
Says it below: ```/* Taken from Wine ntdll/sync.c */```